nav2_mppi_controller/src/controller.cpp

```diff
@@ -92,13 +92,15 @@ geometry_msgs::msg::TwistStamped MPPIController::computeVelocityCommands(
   last_time_called_ = clock_->now();
 
   std::lock_guard<std::mutex> param_lock(*parameters_handler_->getLock());
+  geometry_msgs::msg::Pose goal = path_handler_.getTransformedGoal().pose;
```

Comment:
Shouldn't we get the goal as `transformed_plan.poses.back()`? There's no reason to do another TF operation I don't think and we can find it on L98 below `transformPath()` instead

Reply:
We can't because `transformed_plan` is already pruned.